lib/pcm_stream_channel_modifier/pcm_stream_channel_modifier.c

```diff
@@ -64,6 +72,8 @@ int pscm_zero_pad(void const *const input, size_t input_size, enum audio_channel
 	for (uint32_t i = 0; i < input_size / bytes_per_sample; i++) {
 		if (channel == AUDIO_CH_L) {
 			for (uint8_t j = 0; j < bytes_per_sample; j++) {
+				temp_frame_sum_left += ((*pointer_input) << 8 * j);
+				temp_frame_sum_right += ((*pointer_input) << 8 * j);
```

Comment:
There is a bug in the program which is why this is here. This should be moved to AUDIO_CH_R when bug is fixed